nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.auth-guard.service.js

```diff
@@ -78,6 +78,11 @@ NfRegistryUsersAdministrationAuthGuard.prototype = {
                             // render the logout button if there is a token locally
                             if (self.nfStorage.getItem('jwt') !== null) {
                                 self.nfRegistryService.currentUser.canLogout = true;
+
+                                // Update Registry Configuration following successful login
+                                self.nfRegistryApi.getRegistryConfig().subscribe(function (registryConfig) {
```

Comment:
An unfortunate amount of duplication in this file, maybe it could be refactored in future.